src/main/reactjs/src/redux/sagas/TranslatorDetails.ts

```diff
@@ -0,0 +1,67 @@
+import { all, call, put, takeLatest } from 'redux-saga/effects';
+import { AxiosResponse } from 'axios';
+
+import axiosInstance from 'configs/axios';
+import { ApiEndpoints } from 'enums/api';
+import {
+  ApiTranslatorDetails,
+  PublicTranslatorListApiResponse,
+  TranslatorDetails,
+} from 'interfaces/translator';
+import {
+  TRANSLATOR_DETAILS_ERROR,
+  TRANSLATOR_DETAILS_LOAD,
+  TRANSLATOR_DETAILS_LOADING,
+  TRANSLATOR_DETAILS_RECEIVED,
+} from 'redux/actionTypes/translatorDetails';
+
+const reshapeApiResponse = (
+  details: ApiTranslatorDetails
+): TranslatorDetails => {
+  const languagePairs = details.languagePairs.map(({ fromLang, toLang }) => {
+    return { from: fromLang, to: toLang };
+  });
```

Comment:
Tää osa periaatteessa turha jatkossa, mutta voidaan toki käyttää frontissa `from` ja `to` kenttiä "mistä - mihin" kieliä määrittämään. Ihan tarpeeksi selkeät näinkin.